src/task/script.js

```diff
@@ -389,20 +393,32 @@ function clearUserNotFound() {
 function createSuggestionsList(matches) {
   const listItems = document.getElementById('list-items');
   if (matches.length) {
-    matches.map(({ username }) => {
+    assigneeAvatarLoader.style.display = 'initial';
```

Comment:
Prefer having class name changes over inline style. Controlling style via JavaScript is an expensive task.